kernel/ramfs.c

```diff
@@ -2276,6 +2285,17 @@ static int _fs_ioctl(
             _set_fd_flag(file, 0);
             break;
         }
+        case TIOCSPTLCK:
```

Comment:
Should the `devfs_is_pty_pts_device` check be present for TIOCSPTLCK and TIOCGPTN as well?